File: 1_forloop_add_test/src/forloop_add_test.pas
var
    i, j, toval, sumval: sizeint;

begin
    val(paramstr(1),toval);
    sumval := 0;
    for i:=0 to toval-1 do
        for j:=0 to toval-1 do
            inc(sumval, i+(j xor 1));
    writeln('Pascal: ', sumval);
end.

File: 1_forloop_add_test/src/forloop_add_test.kt
fun main(args: Array<String>) {
    var toval = args[0].toLong()
    var sumval: Long = 0
    for (i in 0..toval-1) {
        for (j in 0..toval-1) {
            sumval += i + (j xor 1)
        }
    }
    println("Kotlin: $sumval");
}
